esp/services/esdl_svc_engine/esdl_binding.cpp

```diff
@@ -2184,8 +2183,6 @@ StringBuffer & EsdlBindingImpl::generateNamespace(IEspContext &context, CHttpReq
 
     ns.appendf("%s%c%s", m_pESDLService->m_serviceNameSpaceBase.str(), m_pESDLService->m_usesURLNameSpace ? '/' : ':', serv);
 
-    if (method && *method)
```

Comment:
This method already includes a condition on the configured namespaceScheme being "scapps". This condition should probably be conditional on it not being "scapps".

Reply:
The _scapps_ style namespace is different even from this behavior. We could implement some configurable behavior (probably extending the range of values of namespaceScheme), but Tony recommended an echo-back categorically because of the complexity of the long-term vision for namespace handling.

Reply:
Based on offline conversation, this change does not appear to be necessary to achieve the intent of the story. Either the change should not be made (because it can affect the namespace returned in the WSDL) or is acceptable (because it is the removal of dead code).

Reply:
You are correct- removing this code was an error. I'm patching it back in.